docs/features/diff-viewer/diff-viewer.md

```diff
@@ -163,6 +164,10 @@ In the `Manage`-view there is a `Comparisons` tab that lists all calculated comp
 
 There you can also see error logs in case a calculation of a comparison failed in the `Details & Logs` dialog.
 
+## Integrate Comparisons in CI Pipeline
+
+See: [Diff Viewer REST API](diff-viewer-rest-api.md).
```

Comment:
A short explanation that there is a REST API which could be used in CI Pipelines to trigger comparisons (just 1 sentence) would be nice here.